packages/ui/src/composites/wui-transaction-list-item/index.ts

```diff
@@ -0,0 +1,83 @@
+import { html, LitElement } from 'lit'
+import { property } from 'lit/decorators.js'
+import '../../components/wui-text/index.js'
+import { elementStyles, resetStyles } from '../../utils/ThemeUtil.js'
+import { customElement } from '../../utils/WebComponentsUtil.js'
+import '../wui-transaction-visual/index.js'
+import styles from './styles.js'
+import { type TransactionType, TransactionTypePastTense } from '../../utils/TypeUtil.js'
+import { type TransactionStatus, type TransactionDirection } from '@web3modal/common'
+
+@customElement('wui-transaction-list-item')
+export class WuiTransactionListItem extends LitElement {
+  public static override styles = [resetStyles, elementStyles, styles]
+
+  // -- State & Properties -------------------------------- //
+  @property() public type: TransactionType = 'approve'
+
+  @property() public descriptions?: string[]
+
+  @property() public date?: string
+
+  @property() public status?: TransactionStatus
+
+  @property() public direction?: TransactionDirection
+
+  @property() public imageURL?: string
+
+  @property() public secondImageURL?: string
+
+  @property({ type: Boolean }) public isNFT?: boolean
+
+  // -- Render -------------------------------------------- //
+  public override render() {
+    const firstDesc = this.descriptions?.[0]
+    const secondDesc = this.descriptions?.[1]
+
+    return html`
+      <wui-flex>
+        <wui-transaction-visual
+          status=${this.status}
+          direction=${this.direction}
+          type=${this.type}
+          .isNFT=${this.isNFT}
+          imageURL=${this.imageURL}
+          secondImageURL=${this.secondImageURL}
+        ></wui-transaction-visual>
+        <wui-flex flexDirection="column" gap="3xs">
+          <wui-text variant="paragraph-600" color="fg-100"
+            >${TransactionTypePastTense[this.type]}</wui-text
+          >
+          <wui-flex class="description-container">
+            ${firstDesc
+              ? html`<wui-text variant="small-500" color="fg-200">
+                  <span>${firstDesc}</span>
+                </wui-text>`
+              : null}
+            ${secondDesc
+              ? html`
```

Comment:
No need nested `html` and might be anti pattern

Reply:
I think you can use `nothing` for this: https://lit.dev/docs/v1/lit-html/writing-templates/#rendering-nothing

Reply:
You can split out into template methods to cleanup a bit.